cmd/machine-config-daemon/pivot.go

```diff
@@ -365,14 +226,20 @@ func run(_ *cobra.Command, args []string) error {
 		changed = true
 	}
 
+	runPivotRebootExists := utils.FileExists(runPivotRebootFile)
```

Comment:
let's not use FileExists here either. Also, the impl for FileExists isn't correct, it returns `true` if for some reason, other errors occur causing pivot to reboot in the middle of an MCD sync (which is what I believe it's happening in an issue I'm looking at)

Reply:
small bikeshed: do we have a `FileExists` that ignores errors (and returns `false`) or "throws" (in golang terms)?  I tend to lean towards the latter, but was actually surprised that the "usually so correct it's painful" [Rust API for this](https://doc.rust-lang.org/std/path/struct.Path.html#method.exists) just swallows errors.